uuid/uuid_test.go

```diff
@@ -88,17 +79,19 @@ func tupleValueIsId(t *testing.T, tuples []interface{}, id uuid.UUID) {
 }
 
 func TestSelect(t *testing.T) {
+	if isUUIDSupported == false {
+		t.Skip("Skipping test for Tarantool without UUID support in msgpack")
+	}
+
 	conn := connectWithValidation(t)
 	defer conn.Close()
 
-	skipIfUUIDUnsupported(t, conn)
-
 	id, uuidErr := uuid.Parse("c8f0fa1f-da29-438c-a040-393f1126ad39")
 	if uuidErr != nil {
 		t.Errorf("Failed to prepare test uuid: %s", uuidErr)
 	}
 
-	resp, errSel := conn.Select(space, index, 0, 1, IterEq, []interface{}{ id })
+	resp, errSel := conn.Select(space, index, 0, 1, IterEq, []interface{}{id})
```

Comment:
Seems this change and below are fixes made by `go fmt`, right?

Reply:
yes

Reply:
I propose to move such changes to a separate PR/commits.

Reply:
separated